### DISCUSSION TITLE
如果将 paopaodns 搭建在公网服务器，如何配合paopao网关实现本地代理；

### DISCUSSION THREAD

**Discussion by gukey** [kkkgo/PaoPaoDNS#174] [想法]
之前是 paopaodns 和网关都在同一局域网，可以添加路由解析出国，如果改为公网的话，出国解析怎么设置好一些，可以这样搭建吗？

**Comment by kkkgo** [comment]
不明白你的描述，但你想实现异地网络互联的话可以在路由器上使用zerotier组建内网然后写静态路由。
